libsrc/apple2/acc_detect_fastchip.s

```diff
@@ -0,0 +1,46 @@
+;
+; Marco van den Heuvel, 2018-05-04
+;
+
+; unsigned char detect_fastchip (void);
+;
+;/* Check for the presence of the Apple IIe Fast Chip slot card.
+; *
+; * Possible return values:
+; * 0x00  : Fast Chip slot card not present
+; * 0x01  : Fast Chip slot card present
+; */
+
+        .export         _detect_fastchip
+        .import         _get_ostype
+
+        .include        "accelerator.inc"
+
+_detect_fastchip:
+        jsr     _get_ostype
+        ldx     #$00
+        cmp     #$30                ; Apple IIe
+        beq     check_slot
+        cmp     #$31                ; Apple IIe enhanced
+        beq     check_slot
+        cmp     #$40                ; Apple IIe Option Card
```

Comment:
I don't see the FastChipIIe being compatible with https://en.wikipedia.org/wiki/Apple_IIe_Card ;-))